subsys/bluetooth/mesh/shell.c

```diff
@@ -2680,6 +2718,8 @@ SHELL_STATIC_SUBCMD_SET_CREATE(mesh_cmds,
 		      cmd_app_key_del, 3, 0),
 	SHELL_CMD_ARG(app-key-get, NULL, "<NetKeyIndex>", cmd_app_key_get, 2,
 		      0),
+	SHELL_CMD_ARG(cmd_network_transmit, NULL, "[<count: 0-7>"
+			" <interval: 10-320>]", cmd_network_transmit, 1, 2),
```

Comment:
This command name doesn't match the others very well. `net-transmit` might be the closest match, but that would be very confusing. Suggestion:

```suggestion
	SHELL_CMD_ARG(transmit-param, NULL, "[<count: 0-7>"
			" <interval: 10-320>]", cmd_network_transmit, 1, 2),
```